### ISSUE TITLE
Mod-added NPC not receiving assets

### ISSUE THREAD

**Issue by Gawdzila** [Synthesis-Collective/SynthEBD#74]
Soooo... SynthEBD has been flawlessly doing its thing for a while now, which makes this new problem I'm seeing extra weird.  After running the patcher to swap meshes and textures on a couple vanilla NPCs, suddenly a companion NPC added by a mod (Daegon) showed up with the blackface issue (unfortunately I have no other mod-added NPCs to check against).  I checked SynthEBD and, looking at the consistency file, found that she did not have any textures being assigned AT ALL.  She appears to be the only one afflicted.

This is especially strange considering that I am manually assigning them in NPC Assignments and I have changed almost nothing since the last time it was working.  The BodySlide seems to be assigned normally, but the assets are just blank. I am forcing the same asset pack as when it was working, and I have re-run the patcher several times, making sure to leave MO and re-load SynthEBD in between.  I have also tried deleting the SynthEBD folder entirely and having the patcher re-build it.  I am running EasyNPC and I have tried it with Daegon both included and not-included in the EasyNPC patch.  SynthEBD appears to produce an NPC record identical to the original mod with the exception that it has an assigned head texture "SkinHeadFemaleHighElf" (the original mod had no head texture assigned in the NPC record).

Any idea what might be going on or what I may have changed by accident?  BTW I am currently running 8.5.0 because it was working well for me and 8.6 had initially crashed.  Do any of the later versions have revisions that might fix this?

**Comment by Piranha91** [comment]
Just want to clarify a few things:

1) Daegon was previous getting SynthEBD textures, and now she is not getting SynthEBD textures? Or she never got SynthEBD textures?
2) Do other NPCs get SynthEBD textures?
3) Daegon's assets are blank in her consistency, or in SSEedit, or both?
4) Which assets do you want Daegon to get?
5) Have you tried adding Daegon to the "Verbose Mode for Specific NPCs" list? If so, what does the verbose log say?

**Comment by Gawdzila** [comment]
1/4/6)  Yes she was previously receiving textures from the ALT2 config, which is what I'd like her to continue using. It doesn't look like any of the destination paths are red, they're all in the green.

2) Yes it looks like all other NPCs are getting textures.  There are some neck seams but I think that's down to texture mismatches or maybe tintmaps, and most NPCs look like they're getting their assigned textures properly.

3) Both, it looks like? In SSEEdit it looks like it just carries over the headparts from the original mod and doesn't add a patched naked armor or head texture, only a vanilla head texture. The original mod just uses whatever replacer you're using, so although it has custom headpart nifs (I believe it uses High Poly Head) it doesn't have non-vanilla texture paths.

5) Ah thanks for reminding me that verbose exists!  I have attached the log; I didn't read the whole thing, but the lines at the top stood out to me:

> _<SpecificAssignments>Subgroup HS.D.Lc requested by Specific NPC Assignment was not found in Asset Pack ALT2 CBBE 4K Compressed
> 					
> 					Subgroup HSp.C.HE requested by Specific NPC Assignment was not found in Asset Pack ALT2 CBBE 4K Compressed
> 					
> 					Subgroup BFS.D.SLC.S requested by Specific NPC Assignment was not found in Asset Pack ALT2 CBBE 4K Compressed
> 					
> 					Subgroup HSu.D.Lc requested by Specific NPC Assignment was not found in Asset Pack ALT2 CBBE 4K Compressed_


The thing is that these DO exist and are in the green (see attached screencap for example), so I can't understand why it isn't assigning them.

![ALT2 Sk Entry](https://user-images.githubusercontent.com/119834693/211395518-eb639eed-cfef-4e3c-beba-2ccf83e62a5b.jpg)

[Daegon (k101Daegon) 005900-k101Daegon.esp.txt](https://github.com/Synthesis-Collective/SynthEBD/files/10376403/Daegon.k101Daegon.005900-k101Daegon.esp.txt)

**Comment by Gawdzila** [comment]
Sooo.... it looks like it is a problem specifically with the ALT2 config, and not only for Daegon.  I switched Daegon to a new config that I made for the Bits n Pieces skin, and it worked -- the consistency entry is populated with assets and it looks fine in-game.  Meanwhile, I gave Camilla Valerius the ALT2 skin as a test, and it also failed to fill in her Consistency entry, and SSEEdit shows that she is simply using a vanilla Imperial head texture.

So I guess the question is, why is the ALT2 config suddenly failing and reporting missing assets when it clearly has them?  I did validate the config and all the boxes look green.
